ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -11,14 +11,10 @@ base_images:
     name: "4.16"
     namespace: ocp
     tag: hypershift-operator
-  ocp_4.14_base:
```

Comment:
@soltysh This is being used by:  https://github.com/openshift/release/blob/master/ci-operator/config/openshift/origin/openshift-origin-master.yaml#L35-L37

Reply:
@droslean this https://github.com/openshift/release/pull/51289 will fix